### ISSUE TITLE
free(): invalid pointer in IDEA plugin example (Linux)

### ISSUE THREAD

**Issue by igordmn** [JetBrains/compose-jb#1494]
Compose 1.0.0-rc5

Crashes at launch (compose-jb/examples/gradlew runIde)

Also in run3, run4, runVSync, runWindowApi in Compose source.

Doesn't crash in run1

**Comment by olonho** [comment]
Doesn't crash with JDK11 on Ubuntu 20.04 with either rc5 or 8bc17fc971e8608c5370677f4f5872dce14cdf8e build for me. Could you please provide more details, i.e. error log or valgrind log.

**Comment by olonho** [comment]
Reproduced on JBR
```
#0  __GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:50
#1  0x00007f6efa6c5859 in __GI_abort () at abort.c:79
#2  0x00007f6efa7303ee in __libc_message (action=action@entry=do_abort, fmt=fmt@entry=0x7f6efa85a285 "%s\n") at ../sysdeps/posix/libc_fatal.c:155
#3  0x00007f6efa73847c in malloc_printerr (str=str@entry=0x7f6efa8584ae "free(): invalid pointer") at malloc.c:5347
#4  0x00007f6efa739cac in _int_free (av=<optimized out>, p=<optimized out>, have_lock=0) at malloc.c:4173
#5  0x00007f6d7b105249 in std::locale::_Impl::_M_install_facet(std::locale::id const*, std::locale::facet const*) ()
   from /home/nike/.skiko/ccef86fd7c93aadb390359415a64a503c1bb3a6e807747dec2e76143309b7a01/libskiko-linux-x64.so
#6  0x00007f6d7b105636 in std::locale::_Impl::_Impl(unsigned long) ()
   from /home/nike/.skiko/ccef86fd7c93aadb390359415a64a503c1bb3a6e807747dec2e76143309b7a01/libskiko-linux-x64.so
#7  0x00007f6d7b1064f5 in std::locale::_S_initialize_once() ()
   from /home/nike/.skiko/ccef86fd7c93aadb390359415a64a503c1bb3a6e807747dec2e76143309b7a01/libskiko-linux-x64.so
#8  0x00007f6efaaba4df in __pthread_once_slow (once_control=0x7f6d7bffbb60 <std::locale::_S_once>, 
    init_routine=0x7f6d7b1064e0 <std::locale::_S_initialize_once()>) at pthread_once.c:116
#9  0x00007f6d7b106541 in std::locale::_S_initialize() ()
   from /home/nike/.skiko/ccef86fd7c93aadb390359415a64a503c1bb3a6e807747dec2e76143309b7a01/libskiko-linux-x64.so
#10 0x00007f6d7b1065a9 in std::locale::locale() ()
   from /home/nike/.skiko/ccef86fd7c93aadb390359415a64a503c1bb3a6e807747dec2e76143309b7a01/libskiko-linux-x64.so
#11 0x00007f6d7b103ac3 in std::ios_base::Init::Init() ()
   from /home/nike/.skiko/ccef86fd7c93aadb390359415a64a503c1bb3a6e807747dec2e76143309b7a01/libskiko-linux-x64.so
#12 0x00007f6d7a6523e0 in _GLOBAL__sub_I_PathEffect.cc ()
   from /home/nike/.skiko/ccef86fd7c93aadb390359415a64a503c1bb3a6e807747dec2e76143309b7a01/libskiko-linux-x64.so
#13 0x00007f6efab17c4a in ?? () from /lib64/ld-linux-x86-64.so.2
#14 0x00007f6efab17d51 in ?? () from /lib64/ld-linux-x86-64.so.2
#15 0x00007f6efa803915 in __GI__dl_catch_exception (exception=<optimized out>, operate=<optimized out>, args=<optimized out>) at dl-error-skeleton.c:182
#16 0x00007f6efab1c20f in ?? () from /lib64/ld-linux-x86-64.so.2
#17 0x00007f6efa8038b8 in __GI__dl_catch_exception (exception=<optimized out>, operate=<optimized out>, args=<optimized out>) at dl-error-skeleton.c:208
--Type <RET> for more, q to quit, c to continue without paging--
#18 0x00007f6efab1b74a in ?? () from /lib64/ld-linux-x86-64.so.2
#19 0x00007f6efa89334c in dlopen_doit (a=a@entry=0x7f6e2d3a2090) at dlopen.c:66
#20 0x00007f6efa8038b8 in __GI__dl_catch_exception (exception=exception@entry=0x7f6e2d3a2030, operate=operate@entry=0x7f6efa8932f0 <dlopen_doit>, 
    args=args@entry=0x7f6e2d3a2090) at dl-error-skeleton.c:208
#21 0x00007f6efa803983 in __GI__dl_catch_error (objname=objname@entry=0x7f6e140131e0, errstring=errstring@entry=0x7f6e140131e8, 
    mallocedp=mallocedp@entry=0x7f6e140131d8, operate=operate@entry=0x7f6efa8932f0 <dlopen_doit>, args=args@entry=0x7f6e2d3a2090)
    at dl-error-skeleton.c:227
#22 0x00007f6efa893b59 in _dlerror_run (operate=operate@entry=0x7f6efa8932f0 <dlopen_doit>, args=args@entry=0x7f6e2d3a2090) at dlerror.c:170
#23 0x00007f6efa8933da in __dlopen (file=<optimized out>, mode=<optimized out>) at dlopen.c:87
#24 0x00007f6ef9dc6284 in os::Linux::dlopen_helper(char const*, char*, int) ()
```

**Comment by akurasov** [comment]
Should be fixed in skiko 0.6.3

**Comment by igordmn** [comment]
Should be fixed in 1.0.0-rc6

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-1494">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
